theano/tensor/nnet/tests/test_nnet.py

```diff
@@ -592,9 +587,9 @@ def test_softmax_grad_optimizations_vector(self):
                 has_softmax = True
             if node.op == softmax_grad:
                 has_softmaxdx = True
-        assert has_cx1hot
+        assert not has_cx1hot
         assert has_cx1hotdx
-        assert not has_softmax
+        assert has_softmax
```

Comment:
We change a crossentropy_softmax_argmax_1hot_with_bias to just a softmax. So we replace a complicated op with multiple output where we use only 1 output by a simpler op that do just the part we do. We where already taking the output that compute the softmax.

Idem for the other case.